package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@gnosis.pm/dex-contracts",
   "version": "0.2.15",
   "description": "Contracts for dFusion multi-token batch auction exchange",
-  "main": "src/index.js",
+  "main": "build/commonjs/index.ts",
```

Comment:
Isn't the path just common and doesn't the file still end in js?

Reply:
The path was wrong, I noticed it when I used `yarn link` with `dex-price-estimator`.